consensus/src/runway/mod.rs

```diff
@@ -146,7 +145,8 @@ where
     rx_consensus: Receiver<NotificationOut<H>>,
     ordered_batch_rx: Receiver<Vec<H::Hash>>,
     finalization_handler: FH,
-    unit_saver: UnitSaver<US, H, D, MK::Signature>,
+    save_unit_tx: mpsc::UnboundedSender<UncheckedSignedUnit<H, D, MK::Signature>>,
+    unit_saved_rx: mpsc::UnboundedReceiver<UncheckedSignedUnit<H, D, MK::Signature>>,
```

Comment:
Make the typing conform to the things around it – the `Sender` and `Receiver` things are just aliases for this iirc. This probably also applies to the other file, but I failed to notice it there. <_<"

Reply:
I also have a preference for naming that tells you what is on the other end of the channel, so something like `units_for_saver` and `units_from_saver`, but that is purely a preference, so if you like the `tx/rx` style more keep these as they are.

Reply:
Good point. I suggest `backup_units_for_saver`, `backup_units_from_saver`. Accordingly we would have `backup_units_for_runway`, `backup_units_from_runway`. Without `backup` prefix, it would be ambiguous what these tx / rx are for.